SPECS/perl-Sub-Install/perl-Sub-Install.spec

```diff
@@ -7,7 +7,7 @@
 
 Name:           perl-Sub-Install
 Version:        0.928
```

Comment:
This one is 10 years old.  There is 1 newer version .929 from 2 years ago.  Consider upgrading this one.